iceoryx_examples/icedocker/README.md

```diff
@@ -32,9 +32,9 @@ Every iceoryx application is registering itself at our central broker RouDi
 by sending a message to the unix domain socket located at
```

Comment:
Line above: `Every iceoryx application registers itself at the central broker RouDi ...`